src/main/webapp/app/markdown-editor/markdown-editor.component.ts

```diff
@@ -88,10 +88,20 @@ export class MarkdownEditorComponent implements AfterViewInit, OnInit {
         return this.previewChild == null;
     }
 
+    /**
+     * @function addCommand
+     * @param command
+     * @desc add a defaultCommand to the view
+     */
```

Comment:
it does not help if you have a comment that is basically the same as the method name. please describe in greater detail what happens here and also why someone would actually invoke this method, i.e. in this case to customize the user experience of the markdown editor